source/User_Guide/Marketing_Campaigns/templates.md

```diff
@@ -79,3 +79,6 @@ Creating a Template from a Campaign
 1. From the left-hand navigation, select **Marketing Campaigns**, and then click **Campaigns**.
 1. Find the campaign you would like to use, click the **action menu**, and then select **Create Template**. 
 
+{% info %}
+For information on Sending Campaigns, visit our [Sending an Email](https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/getting_started.html) article or [view the next video](https://vimeo.com/139274837) in our Getting Started Series.
```

Comment:
Change to:
For more information, see [Sending an Email] or [view the next video] in our Getting Started series.